client/state/ui/section/test/reducer.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { SECTION_SET } from 'state/action-types';
+import reducer, {
+	name,
+	paths,
+	module as moduleReducer,
+	enableLoggedOut,
+	secondary,
+	group,
+	isomorphic,
+	title
+} from '../reducer';
+
+describe( 'reducer', () => {
+	it( 'should include expected keys in return value', () => {
+		expect( reducer( undefined, {} ) ).to.have.keys( [
```

Comment:
>any other addition to `section` later on will automatically require this particular test to be kept up-to-date.

That's the intent, yeah. I think as-is that's captured? For example, I find commenting one of these keys in the test will cause it to fail.